pkg/skaffold/watch/triggers.go

```diff
@@ -109,3 +114,51 @@ func (t *manualTrigger) Start() (<-chan bool, func()) {
 
 	return trigger, func() {}
 }
+
+// notifyTrigger watches for changes when fsnotify
+type fsNotifyTrigger struct {
+}
+
+// Debounce tells the watcher to not debounce rapid sequence of changes.
+func (t *fsNotifyTrigger) Debounce() bool {
+	return false
+}
+
+func (t *fsNotifyTrigger) WatchForChanges(out io.Writer) {
+	color.Yellow.Fprintln(out, "FS Notify starting")
```

Comment:
This message is for the user to understand what's going on.
Should be something like `Watching for changes...`